### PR TITLE
update zh translation /docs/reference/setup-tools/kubeadm/generated/k…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_token_create.md [hunk 1]
```diff
@@ -48,122 +41,111 @@ kubeadm token create [token]
       <td colspan="2">--config string</td>
     </tr>
     <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">Path to kubeadm config file (WARNING: Usage of a configuration file is experimental)</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      Path to a kubeadm configuration file.
+      -->
+      指定 kubeadm 配置文件的路径。
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_token_create.md]:
```suggestion
      到 kubeadm 配置文件的路径。
```

Comment (tanjunchen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_token_create.md]: 好的

**Comment by tanjunchen** [comment]
@xichengliudui  感谢审查，已经修改

**Pull request merged**